save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/config/LocalDateTimeConfig.kt

```diff
@@ -0,0 +1,66 @@
+@file:Suppress(
```

Comment:
May be add some comments, that this file should be in consistent with the same from preprocessor?

Reply:
Added